online.codingblocks.com/components/Base/CourseCard.vue

```diff
@@ -166,6 +166,9 @@ export default {
   border-radius: 50%;
 }
 
+.twoLine{
```

Comment:
why?

Reply:
Didn't do this intentionally. Maybe switching between master and development branch created some problem

Reply:
you would need to rebase or cherry-pick the relevant commits. In the current state, I cannot. merge this